lm75bd/lm75bd.c

```diff
@@ -9,6 +9,7 @@
 
 /* LM75BD Registers (p.8) */
 #define LM75BD_REG_CONF 0x01U  /* Configuration Register (R/W) */
+#define LM75BD_REG_PTR  0x00
```

Comment:
0x00 corresponds to the temperature register. Similar to the config register, specify whether it's read-only or if writes are allowed.

```suggestion
#define LM75BD_REG_TEMP  0x00 /* Temperature Register (R) */
```